>From bbfc42165b836c22bff106eddab9f2327788568c Mon Sep 17 00:00:00 2001
From: Sebastian Schmidt <yath@yath.de>
Date: Sat, 17 May 2014 06:57:47 +0200
Subject: [PATCH] Fix screen width detection

3163087b moved SigWinch(int) from apt-get.cc to private-output.cc
without moving #include <sys/ioctl.h>, making SigWinch a nop.
---
 apt-private/private-output.cc | 1 +
 cmdline/apt-get.cc            | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/apt-private/private-output.cc b/apt-private/private-output.cc
index 7f3eef6..32f27a3 100644
--- a/apt-private/private-output.cc
+++ b/apt-private/private-output.cc
@@ -23,6 +23,7 @@
 #include <langinfo.h>
 #include <unistd.h>
 #include <signal.h>
+#include <sys/ioctl.h>
 
 #include <apti18n.h>
 									/*}}}*/
diff --git a/cmdline/apt-get.cc b/cmdline/apt-get.cc
index 566103f..0f18b0e 100644
--- a/cmdline/apt-get.cc
+++ b/cmdline/apt-get.cc
@@ -76,7 +76,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/ioctl.h>
 #include <sys/stat.h>
 #include <sys/statfs.h>
 #include <sys/statvfs.h>
-- 
2.0.0.rc2

